test/llvm/vectorization/simple_loop.compopts

```diff
@@ -0,0 +1 @@
+--llvm --fast --vectorize --llvm-print-ir loop --llvm-print-ir-stage full --mllvm -force-vector-width=4 --mllvm -force-vector-interleave=1
```

Comment:
GitHub is showing a funny line ending here, maybe, there's an extra newline or something?